unitTests/ui/InitializationTest.ts

```diff
@@ -399,6 +399,154 @@ export function InitializationTest() {
       Simulate.removeJQuery();
     });
 
+    describe('when initializing a Search Interface', () => {
+      let spyAfterComponentsInitialized: jasmine.Spy;
+      const doInit = () => {
+        return Initialization.initializeFramework(root, searchInterfaceOptions, () => {
+          return Initialization.initSearchInterface(root, searchInterfaceOptions);
+        });
+      };
+
+      beforeEach(() => {
+        spyAfterComponentsInitialized = jasmine.createSpy('afterComponentsInitialized');
+        $$(root).on(InitializationEvents.afterComponentsInitialization, spyAfterComponentsInitialized);
+      });
+
+      afterEach(() => {
```

Comment:
There are several `afterEach` blocks in this file that are setting variables to `null`. However, in the `beforeEach`, these variables are reinitialized, so is there a need for the `afterEach` blocks?

Reply:
In that case, I would remove the `afterEach` :)
No point in keeping code does not serve a purpose,